pipeline/start_server.py

```diff
@@ -20,6 +20,8 @@
 from argparse import ArgumentParser
 from pathlib import Path
 
+from huggingface_hub import get_token
```

Comment:
Can you please add a try/except here and define get_token to just return os.getenv("HF_TOKEN", "") in case it cannot be imported (with some warning if you'd like)? We try to keep the pipeline scripts running without any extra packages installed as they are run on slurm clusters and it's best not to install extra things there

Reply:
Good point will do.